### PR TITLE
feat(GP-13): relax update constraint

### PR DESCRIPTION
**Pull request by khan08**
Description about what this pull request does.

Please make sure to follow the [DEV guidelines](https://gen3.org/resources/developer/dev-introduction/) before asking for review.

### New Features
- Implemented XXX

### CODE REVIEW

File: indexd/index/drivers/alchemy.py [hunk 1]
```diff
@@ -808,6 +808,10 @@ def update(self, did, rev, changing_fields):
                     )
                     for m_key, m_value in changing_fields['metadata'].items()]
 
+            if 'hashes' in changing_fields:
```
Comment (singergr) [inline indexd/index/drivers/alchemy.py]: Should we make this work more like updates to metadata/urls_metadata and delete any hashes that aren't given in the request? Like if the record already has SHA-256 and MD5, and the client just passes a new MD5, should we delete the old SHA-256?

File: indexd/index/schema.py [hunk 2]
```diff
@@ -147,6 +147,9 @@
         "file_name": {
             "type": ["string", "null"],
         },
+        "size": {
+            "type": "integer",
+        },
```
Comment (singergr) [inline indexd/index/schema.py]: Might want to set the minimum to 0 like we do for POST.

File: indexd/index/drivers/alchemy.py (hunk 1 above)
Comment (khan08) [reply indexd/index/drivers/alchemy.py]: I started with overwrite, then switch to update. Thinking about it more I think I should make the behavior consistent with the metadata/urls_metadata.
TLDR I'm changing this to delete and overwrite.
